src/metabase/api/public.clj

```diff
@@ -16,7 +16,7 @@
              [dashboard-card :refer [DashboardCard]]
              [dashboard-card-series :refer [DashboardCardSeries]]
              [field-values :refer [FieldValues]]]
-            [metabase.query-processor.expand :as ql]
+            [metabase.query-processor.middleware.expand :as ql]
```

Comment:
normally for a long-running PR like this if you do significant refactoring I would usually try to pull things like renaming a namespace into a separate PR and merge that ASAP, and then rebase the PR on master once it's merged. It makes it easier to see the actual changes that go on vs hundreds of "changes" that are nothing more than a renamed file. Not sure if this is the most efficient way to do it but I usually do

```bash
git checkout master
git pull
git merge <branch>
git reset origin/master
```

which will give you all the changes in the branch in an uncommited state. Then I usually go back and toss everything besides the "code cleanup"/refactoring and submit that as a PR.

(For next time)